demo/runners/support/agent.py

```diff
@@ -1286,6 +1322,22 @@ async def start_endorser_agent(start_port, genesis):
     log_msg("Endorser Endpoint URL is at:", endorser_agent.endpoint)
     log_msg("Endorser webhooks listening on:", start_port + 2)
 
+    # get a reusable invitation to connect to this endorser
+    log_msg("Generate endorser multi-use invite ...")
+    endorser_agent.endorser_connection_id = None
+    endorser_agent.endorser_public_did = None
+    endorser_connection = await endorser_agent.admin_POST(
+        "/connections/create-invitation?alias=EndorserMultiuse&auto_accept=true&multi_use=true"
+    )
```

Comment:
Would it make sense to use OOB invitations (also on the receiver end) rather than connection invitations? My understanding was that at some point it would replace connection invitation completely and therefore should be preferred - @shaangill025 probably has better/fresher information on this.

Reply:
Yes good observation, I'll make the change

Reply:
It now checks the invitation type and supports either OOB or "old-style" connections